mplc/experiment.py

```diff
@@ -20,18 +21,23 @@ def __init__(
             self,
             experiment_name=None,
             nb_repeats=1,
+            scenarios_list=[]
     ):
         """
         :param experiment_name: string, name of the experiment
         :param nb_repeats: int, number of repeats of the experiments (as an experiment includes
                            a number of non-deterministic phenomena. Example: 5
+        :param scenarios_list: list, list of scenarios to be run during the experiment. scenario can also be added via
         """
 
         self.name = experiment_name
-        self.scenarios_list = []
-        self.nb_repeats = nb_repeats
         if experiment_name:
             self.experiment_path = self.define_experiment_path()
+        self.scenarios_list = []
+        for scenario in scenarios_list:
+            self.add_scenario(scenario)
```

Comment:
I am not too fan of that solution. Because now a user could (naively, but realistically) do `my_exp.scenarios_list.append(my_new_scenario)`, and in that case your `save_folder` will not be initialized properly.

Reply:
I will make the scenarios_list private, even if python has a strange way of considering the private aspect of private attributes